coda/coda_validate/models.py

```diff
@@ -1,7 +1,36 @@
 from django.db import models
 from datetime import datetime
 
-# Create your models here.
+
+class ValidateManager(models.Manager):
+
+    def last_verified_status_counts(self):
+        results = self._verified_counts()
+        return VerifiedCountsResultFormatter(results).format()
+
+    def _verified_counts(self):
+        return (self.values('last_verified_status')
+                    .annotate(count=models.Count('last_verified_status')))
```

Comment:
I noticed it on `/validate/` and then `/validate/check.json`.
I believe I stopped the containers, removed the app container, and then built again.